config/matching_params.yml

```diff
@@ -163,6 +166,7 @@ elevation_keys:
   - Höhe
   - Kote
   - OK Terrain
+  - Tiefe ab OKT
```

Comment:
Depth is something different from elevation, this does not seem to belong here. For which input does this bring an improvement?

Reply:
I agree. This used to make a difference at some point. However, now I checked again and I cannot see any effect anymore. Thank you for making me look into it a second time after I added all the changes.